src/components/pages/Verify/SubmitVerification.tsx

```diff
@@ -0,0 +1,150 @@
+  constructor(props: SubmitVerificationProps) {
+    super(props);
+  }
+
+  submitVerification = () => {
+    this.props.verify(this.props.toMemberId, this.props.videoToken);
+  };
+
+  private _renderSubmittingStatus = () => {
+    const statusType = this.props.submitVerificationStatus
+      ? this.props.submitVerificationStatus.status
+      : undefined;
+    switch (statusType) {
+      case ApiCallStatusType.STARTED:
+        return <Text style={styles.text}>Submitting verification...</Text>;
+      case ApiCallStatusType.SUCCESS:
+        return (
+          <React.Fragment>
+            <Text style={styles.text}>Verification successful!</Text>
+            <Button
+              style={styles.button}
+              title="Return"
+              onPress={this.props.onExit}
+            />
+          </React.Fragment>
+        );
+      case ApiCallStatusType.FAILURE:
+        return <Text style={styles.text}>Verification failed.</Text>;
+      default:
+        return undefined;
+    }
+  };
+
+  render() {
+    const status = this.props.submitVerificationStatus
+      ? this.props.submitVerificationStatus.status
+      : undefined;
+    const isRequestSendingOrSent =
+      status &&
+      (status === ApiCallStatusType.STARTED ||
+        status === ApiCallStatusType.SUCCESS);
+    return (
+      <View style={styles.container}>
+        <Text style={styles.back} onPress={this.props.onBack}>
+          Back
+        </Text>
+        <View style={styles.card}>
+          <Text style={styles.text}>
+            By clicking the "Verify" button below, you are confirming that you
+            believe that this account belongs to{" "}
+            <Text style={styles.name}>{this.props.toMemberFullName}</Text> and
+            this is the only time they have joined Raha.
+          </Text>
+          <Button
+            title="Verify"
+            onPress={this.submitVerification}
+            disabled={isRequestSendingOrSent}
+            style={styles.button}
+          />
+          {this._renderSubmittingStatus()}
+        </View>
+      </View>
+    );
+  }
+}
+
+const mapStateToProps: MapStateToProps<ReduxStateProps, OwnProps, RahaState> = (
+  state,
+  ownProps
+) => ({
+  submitVerificationStatus: getStatusOfApiCall(
+    state,
+    ApiEndpointName.VERIFY_MEMBER,
+    ownProps.toMemberId
+  )
+});
+
+export const SubmitVerification = connect(
+  mapStateToProps,
+  { verify }
+)(SubmitVerificationView);
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: colors.darkBackground,
+    flex: 1,
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center"
+  },
+  card: {
+    backgroundColor: colors.lightBackground,
+    width: Dimensions.get("window").width - 24,
+    padding: 12,
+    borderRadius: 12
+  },
+  text: {
+    fontSize: 18,
+    marginVertical: 4,
+    marginHorizontal: 40,
+    textAlign: "center"
+  },
+  name: {
+    fontWeight: "bold"
```

Comment:
same goes for other occurrences of this; though it's ok if you don't feel like fixing it, with respect to both this and font sizes, I will do a pass later to make them all consistent by using the `fontSizes` listed in helpers instead of ad hoc like this, since you're not the only offender.

Reply:
Agreed, I think for both this and uses of the "R" symbol, we should just do an all-at-once pass, since we're probably doing both all over the place.

Reply:
https://github.com/rahafoundation/raha-mobile-app/issues/202

Reply:
https://github.com/rahafoundation/raha-mobile-app/issues/201